bears/general/ImageDimensionBear.py

```diff
@@ -0,0 +1,53 @@
+import os
+import yaml
+
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.results.Result import Result
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+from dependency_management.requirements.GemRequirement import GemRequirement
+from dependency_management.requirements.PipRequirement import PipRequirement
+from sarge import run, Capture
+
+
+class ImageDimensionBear(GlobalBear):
+    """
+    Checks the dimension of an image.
+
+    More information is available at <github.com/Abhi2424shek/img_checker>
+    """
+
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    REQUIREMENTS = {GemRequirement(' img_checker'),
```

Comment:
ok, I have submitted a PR for that https://gitlab.com/coala/package_manager/merge_requests/10